.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+# Problem
+
+An overview of the background required to understand the problem.
+A problem description.
+
+# Implementation
```

Comment:
We can call it "Solution".

Reply:
Well, the idea was not about providing solution in the issue but rather about some known implementation details and unknown paths to be explored.

Reply:
But implementation is a solution of a problem :P I just wanted to be more generic but maybe this will give a more clear hint what to actually write here.

Reply:
Let's see. I'm not against "solution" if we're going to describe solutions there. A solution, I think, is what you produce as a PR and it's not a solution until the PR is in place, I think, it's something like "Implementation Notes" which is just "Implementation". Something like that.